__init__.py

```diff
@@ -704,7 +690,8 @@ def validate_duration(string):
         wait_while_speaking()
         self.enable_intent("handle_mute_timer")
         # Start showing the remaining time on the faceplate
-        self.update_display(None)
+        if self.platform != "mycroft_mark_2":
+            self.update_display(None)
```

Comment:
instead of passing None here, can we just have a default arg in the update_display function directly? a common pattern for handlers that trigger by bus messages and manually is just passing message=None as default arg

Reply:
If you see the check clearly we don't use self.update_display(...) at all anymore because for gui again qml.handles the timers and updates, the above if wanted changed for mark-1 should be another issue since.

Reply:
i get that, what im saying is the check should be

```
if self.platform == "mycroft_mark_1":
    self.update_display()
```

and then
```
def update_display(message=None):
    ....
```
or if its not even triggered by messagebus anymore, then just remove the argument

just feels weird passing the None on the function call